src/Searcher.h

```diff
@@ -9,12 +9,15 @@
 
 #include <algorithm>
 #include <vector>
+#include <queue>
 
 /**
  * A class that performs alpha-beta search in the game of Domineering to find
  * the best possible move for the current turn.
  */
 
+template<typename T> using branch_queue = std::priority_queue<Node, std::vector<Node>, T>;
```

Comment:
I don't think you're using this. If you don't plan on using it, might as well delete this and `#include <queue>`.